### PR TITLE
Add yapf recipe.

### PR DESCRIPTION
**Pull request by vamega**

### CODE REVIEW

File: recipes/yapf/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    - python
+
+test:
+  commands:
+    - yapf --help
+    - yapf --version
+
+about:
+  home: https://github.com/google/yapf
+  license: Apache Software License 2.0
```
Comment (msarahan) [inline recipes/yapf/meta.yaml]: Could shorten this to simply:

Apache 2.0

File: recipes/yapf/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+  summary: 'A formatter for Python files'
```
Comment (msarahan) [inline recipes/yapf/meta.yaml]: Quotes not necessary

**Comment by msarahan** [comment]
Very nice, clean recipe.  Minor comments - will let this sit a bit for comments from others.  I am +1 to merge.

**Comment by vamega** [comment]
Updated after fixing all the comments.

**Comment by ocefpaf** [comment]
+1
Em 14/06/2016 12:45, "Varun Madiath" <notifications@github.com> escreveu:

**Comment by vamega** [comment]
Do I need to do something now? I'm not completely clear on how to proceed from here.
Is this the point where I need the conda-smithy tool?

**Comment by msarahan** [comment]
Nope, you have done everything you need to. Once we merge this, the rest is automated
